rosidl_generator_rs/resource/srv.rs.em

```diff
@@ -0,0 +1,37 @@
+@{
+for subfolder, service in srv_specs:
+    msg_specs.append((subfolder, service.request_message))
+    msg_specs.append((subfolder, service.response_message))
+
+TEMPLATE(
```

Comment:
Wouldn't it be sufficient to only generate the `service.request_message` and `service.response_message` here and not regenerating all of the messages?

Reply:
Not sure I'm following, but you also need a reference to the service type support, hence the rest of the structures in this template.

Reply:
You append the `srv_specs` to `msg_specs` and then re-instantiate the `msg.rs.em` template with the extended `msg_specs`. Instead, couldn't you instantiate it only for `srv_specs`?

Reply:
Ah, you're right, good catch! I don't remember why I used `msg_specs`, I guess I assumed it was empty, anyway fixed in b0fb8ca